scriptmodules/system.sh

```diff
@@ -84,7 +84,7 @@ function get_os_version() {
                 __os_release=10
             fi
 
-            if compareVersions "$__os_debian_ver" lt 8; then
+            if compareVersions "$__os_debian_ver" lt 9; then
```

Comment:
This was actually intentional - I will need to revert this part. Although we don't support versions less than 9, we don't stop retropie-setup from launching (the main setup script has an additional warning).